cypress/integration/basic-auth.spec.ts

```diff
@@ -0,0 +1,14 @@
+import { datasetPage } from "../page-objects/app/datasets-page";
+
+describe("Basic browser auth", () => {
+
+  it("should login into webApp", () => {
+    datasetPage.visit();
+    datasetPage.leftPanel.leftPanelHeader;
+  });
+
+  it("type in the input the name of a dataset", () => {
+    datasetPage.visit();
+    datasetPage.leftPanel.searchInput.type('xl_PGP3140_wgs_NIST-4_2');
```

Comment:
I understand it's a test just for checking everything works fine. But let's add assertions in the future